firmware/raspberry_pi4_network_boot_beta.md

```diff
@@ -40,9 +40,11 @@ sudo echo FIRMWARE_RELEASE_STATUS="beta" > /etc/default/rpi-eeprom-update
 
 ### Configuration - Enable network boot
 Network boot is not enabled by default in the bootloader. To enable it the bootloader configuration file must be edited.
+
+**NOTE:** The `/lib/firmware...` line below ends with a date at the end of the file's name. There are several versions of the beta bootloader in there, so make sure to select the most up-to-date version for your Raspberry Pi.
 ```
 # Extract the configuration file
-cp /lib/firmware/raspberrypi/bootloader/beta/pieeprom-2019-09-23.bin pieeprom.bin
+cp /lib/firmware/raspberrypi/bootloader/beta/pieeprom-$DATE.bin pieeprom.bin
```

Comment:
I think $DATE will not do the what the user expects if this is pasted because it will be expanded by the shell into either the wrong date or the empty string. Perhaps pieeprom-LATEST.bin would be obvious enough

Reply:
This is that fun gray area of "whats the best way to convey this information to a new user".... 

I'm OK with whatever you think would be the best way to convey the intended information.

Reply:
...or `pieeprom-YYYY-MM-DD.bin` :man_shrugging: